src/services/refactors/extractMethod.ts

```diff
@@ -582,34 +568,33 @@ namespace ts.refactor.extractMethod {
         }
     }
 
-    function getUniqueName(isNameOkay: (name: string) => boolean) {
+    function getUniqueName(fileText: string): string {

```

Comment:
So, somewhat ironically, if you were to add a comment `//TODO newFunction_1 goes here`, we would call the new function "newFunction_2"?

Reply:
Would it be easier to use something temporary, like a GUID, until the replacement has been accomplished and then swap it for the name we would originally have generated?

Reply:
That might be doable, but we might (might not) need the GUID to have an identical length. But you could still calculate them together inside `getUniqueName`.